README.md

```diff
@@ -3,99 +3,55 @@
 [![Build Status](https://travis-ci.org/devops-s17-payments/payments.svg?branch=master)](https://travis-ci.org/devops-s17-payments/payments)
 [![codecov](https://codecov.io/gh/devops-s17-payments/payments/branch/master/graph/badge.svg)](https://codecov.io/gh/devops-s17-payments/payments)
 
-**Current payment options include:**
-* Credit Card
-* Debit Card
-* Paypal
 
-**Payment Method**
-```
-{
-    "charge-history": 0.0,
-    "default": false,
-    "detail": {
-        "expires": "08/2018",
-        "name": "Jane Jenkins",
-        "number": "9876543212345678",
-        "type": "Visa"
-    },
-    "id": 2,
-    "nickname": "my-debit",
-    "type": "debit"
-
-}
-```
+Check us out on Bluemix: http://nyu-devops-sp17-payments.mybluemix.net
 
-**How to use this API**
+and here's the containerized version: https://nyu-devops-sp17-payments-docker.mybluemix.net/
 
-**CREATE** a payment method:
- * POST /payments
- * Accepts JSON in request body in following format:
- 
-```
- {
-    "detail":
-    {
-        "expires": "08/2018",
-        "name": "Jane Jenkins",
-        "number": "9876543212345678",
-        "type": "Visa"
-    },
-    "nickname": "my-debit",
-    "type": "debit"
-}
-```
+API Documentation thanks to [Flasgger](https://github.com/rochacbruno/flasgger "Flasgger")
+
+http://nyu-devops-sp17-payments.mybluemix.net/apidocs/index.html
+
+
+### How to use this repository ###
 
-**RETRIEVE** a payment method:
- * GET /payments/{id}
+First, you need some tools. Download [VirtualBox](https://www.virtualbox.org/ "VirtualBox") and [Vagrant](https://www.vagrantup.com/ "Vagrant")
 
-**UPDATE** a payment method:
- * PUT /payments/{id}
- * Accepts JSON in request body. Need to provide entire object as with CREATE.
+Next, export these environment variables *on your machine* so the Docker Container can use them to create the database:
 
 ```
- {
-    "detail":
-    {
-        "expires": "08/2018",
-        "name": "Jane Jenkins",
-        "number": "9876543212345678",
-        "type": "Visa"
-    },
-    "nickname": "my-new-debit",
-    "type": "debit"
-}
+export DB_USER=payments
+export DB_PASSWORD=payments
+export DB_NAME=dev
+export LOCAL_DB=postgresql://payments:payments@localhost:5432/dev
```

Comment:
I think this should be `LOCAL_DB=postgresql://payments:payments@payments-database:5432/dev`.